ext/vscode/package.nls.json

```diff
@@ -58,6 +58,9 @@
     "azure-dev.utils.quickPickWorkspaceFolder.title": "Open Folder",
     "azure-dev.utils.spawn.exited": "Process '{0}' exited with code {1}",
     "azure-dev.utils.spawn.exitError": "\nError: {0}",
+    "azure-dev.utils.azd.notInstalled": "Azure Developer CLI is not installed. Visit {0} to get it.",
```

Comment:
Maybe we should clarify here to say something like:

"This extension depends on having the Azure Developer CLI (azd) installed on your computer. Visit <link> to install it."

(I'm flexible on wording).

This was a common issue with the core Python extension where people thought that by installing the extension they were also getting a version of Python installed.

Reply:
Great point. Reworded.